packages/contact-exchange/src/tests/HandleOOBInvitation.test.ts

```diff
@@ -0,0 +1,49 @@
+// handleOOBInvitation.test.ts
+import { handleOOBInvitation } from '../services/HandleOOBInvitation';
+import { Wallet } from '../services/Wallet';
+import { parseOOBInvitation } from '../services/OOBParser';
+import {
+  validOutOfBandInvitation,
+  validEncodedUrl,
+  invalidEncodedUrl,
+  invalidOutOfBandInvitation,
+} from '../services/OOBTestFixtures';
+
+describe('handleOOBInvitation', () => {
+  let wallet: Wallet;
+
+  beforeEach(() => {
+    wallet = new Wallet();
+  });
+
+  it('should add a valid contact to the wallet from a JSON invitation', () => {
+    handleOOBInvitation(wallet, validOutOfBandInvitation, 'wallet-1');
+    const contacts = wallet.getContacts('wallet-1');
+
+    expect(contacts).toHaveLength(1);
+    expect(contacts[0].did).toBe('did:example:123456789abcdefghi#key-1');
+  });
+
+  it('should add a valid contact from a base64 encoded URL invitation', () => {
+    const parsedInvitation = parseOOBInvitation(validEncodedUrl);
```

Comment:
Since these tests are just testing the parseOOBInvitation inside the OOBParser file, can you please place them in a specific test file so it is easier to know where to go when we want to match tests and methods?